Vialer/AppDelegate.m

```diff
@@ -334,6 +334,7 @@ - (void)handleSipCall:(GSCall *)sipCall {
 - (void)showLogin {
     if (!self.loginViewController.presentingViewController) {
         // Set animated to NO to prevent a flip to the login/onboarding view.
+        //self.window.rootViewController = self.loginViewController;
```

Comment:
Please remove code that you don't need. Don't introduce unnecessary commented code, please.